docs/content/tutorials/database-ingress/README.md

```diff
@@ -0,0 +1,59 @@
+# Database Load Balancing
+
+NGINX Ingress Controller supports load balancing arbitrary TCP and UDP services using a Custom Resource Definition called a TransportServer.
+
+## Usage
+
+This directory contains a sample using redis; this can easily be adapted to other services.
+
+## Installing NGINX Ingress Controller
```

Comment:
We should set the pattern of referring to one set of installation documents.
That frees the tutorial to focus on describing the use case and the configuration of NIC and any backing service.

If coffee-tea is used, then we should refer to one set of instructions to deploy that service as well.

If we need the enhance the installation doc then we should.
If we need to call out specific gotchas to make sure the customer has done something, that is fine to be here.